source/modulo_components/modulo_components/component_interface.py

```diff
@@ -250,7 +250,17 @@ def __validate_parameter(self, parameter: sr.Parameter) -> bool:
             if value <= 0 or value > sys.float_info.max:
                 self.get_logger().error("Value for parameter 'rate' has to be a positive finite number.")
                 return False
-        return self.on_validate_parameter_callback(parameter)
+        try:
+            result = self.on_validate_parameter_callback(parameter)
+        except AttributeError:
+            self.get_logger().error("Attribute error during 'on_validate_parameter_callback', declare necessary "
+                                    "attributes before __init__ of the parent class")
+            raise
+        if result is None:
+            self.get_logger().error("'on_validate_parameter_callback' doesn't return a value, "
+                                    "parameter change will be rejected.")
```

Comment:
combination of the previous two comments;
```suggestion
            self.get_logger().error("Expected a return value from 'on_validate_parameter_callback'. "
                                    "The parameter change will be rejected.")
```